support-frontend/assets/pages/contributions-landing/components/ContributionFormFields.tsx

```diff
@@ -22,22 +23,20 @@ import {
 } from '../contributionsLandingActions';
 import type { State } from '../contributionsLandingReducer';
 import ContributionState from './ContributionState';
-import '../contributionsLandingReducer';
-import { TextInput } from '@guardian/src-text-input';
+
 // ----- Types ----- //
 
-/* eslint-disable react/no-unused-prop-types */
 type PropTypes = {
 	firstName: string;
 	lastName: string;
 	email: string;
 	billingState: StateProvince | null;
 	checkoutFormHasBeenSubmitted: boolean;
 	isSignedIn: boolean;
-	updateFirstName: (arg0: Event) => void;
-	updateLastName: (arg0: Event) => void;
-	updateEmail: (arg0: Event) => void;
-	updateBillingState: (arg0: Event) => void;
+	updateFirstName: (firstName: string) => void;
+	updateLastName: (lastName: string) => void;
+	updateEmail: (email: string) => void;
+	updateBillingState: (billingState: string) => void;
```

Comment:
I think it's a nicer interface if these just take strings rather than the entire event object

Reply:
Agreed, this is so much easier to read